tsconfig.json

```diff
@@ -10,6 +10,9 @@
         "outDir": "./dist",
         "declaration": true,
         "sourceMap": true,
-        "lib": ["es2015", "dom"] // I need DOM because of typedoc..
```

Comment:
Actually `ChromeDevToolFormatters` use `window` so I propose removing the comment